crossbeam-skiplist/src/base.rs

```diff
@@ -881,7 +886,7 @@ where
 
                 // Write the key and the value into the node.
                 ptr::write(&mut (*n).key, key);
-                ptr::write(&mut (*n).value, value);
+                ptr::write(&mut (*n).value, value());
```

Comment:
I would prefer to create the value before creating a node, as closure may panics.

https://github.com/crossbeam-rs/crossbeam/blob/220e690d1d5d0643cfc006aa10455bd7b28297ef/crossbeam-skiplist/src/base.rs#L878-L879

Reply:
I have created new PR request on latest codebase. Please review that at the earliest.